src/openforms/emails/templates/emails/admin_digest.html

```diff
@@ -31,6 +31,34 @@ <h5>{% trans "Registrations" %}</h5>
                         View failed '{{ form_name }}' submissions
                     {% endblocktranslate %}
                 </a>
+                {% if failed_plugin_data.error %}
+                    <i>* {% trans "Consider enabling outgoing request logs from the configuration in order to investigate the network traffic." %}</i>       
+                {% endif %}
+            </li>
+        {% endfor %}
+    </ul>
+{% endif %}
+
+{% if failed_prefill_plugins %}
+    <h5>{% trans "Prefill plugins" %}</h5>
+    <ul>
+        {% for failed_plugin, failed_plugin_data in failed_prefill_plugins.items %}
+            <li>
+                {% blocktranslate with counter=failed_plugin_data.counter initial_failure_at=failed_plugin_data.initial_failure_at|time:"H:i" last_failure_at=failed_plugin_data.last_failure_at|time:"H:i" trimmed %}
```

Comment:
```suggestion
                {% blocktranslate with counter=failed_plugin_data.counter initial_failure_at=failed_plugin_data.initial_failure_at|time:"TIME_FORMAT" last_failure_at=failed_plugin_data.last_failure_at|time:"TIME_FORMAT" trimmed %}
```

So that it can be localized?